components/contribution-flow/StepDetailsCrypto.js

```diff
@@ -55,18 +98,30 @@ const StepDetailsCrypto = ({ onChange, data }) => {
         {intl.formatMessage(messages['donationAmount'])}
       </Label>
       <StyledInputGroup
-        prepend={currencyType.labelWithoutImage}
+        prepend={cryptoCurrencyType.labelWithoutImage}
         type="number"
         inputMode="decimal"
         defaultValue={amount}
         onChange={({ target }) => {
           setAmount(target.value);
-          dispatchChange('amount', parseFloat(target.value));
+          if (target.value > 0) {
```

Comment:
Why not parseFloat before the test?

Reply:
Makes sense, I've added `parseFloat` at the start of the `onChange` function.